open-budget/app/scripts/analysis_header.coffee

```diff
@@ -41,6 +41,8 @@ class BudgetAnalyzer extends HeaderAnalyzer
 class CurrentBudgetAnalyzer extends BudgetAnalyzer
 
     shouldParticipate: () ->
+        # make sure currentItem exists - /equivs api can return an empty result
+        window.pageModel.get('currentItem')? and
```

Comment:
I believe that coffeescript allows this as well:

window.pageModel.get('currentItem')?.attributes.net_allocated?

(just fyi)